package.json

```diff
@@ -108,10 +108,11 @@
     "@angular/platform-browser-dynamic": "^8.0.0",
     "@angular/router": "^8.0.0",
     "@angular/upgrade": "^8.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^5.3.0",
     "angular": "^1.7.9",
     "angular-cookies": "^1.7.9",
     "angular-route": "^1.7.9",
-    "bootstrap": "^3.4.1",
+    "bootstrap": "^4.0.0",
```

Comment:
```suggestion
    "bootstrap": "^4.5.0",
```

Reply:
Changed it to 4.3.1 based on this: 
<img width="1440" alt="Screenshot 2020-07-05 at 7 57 15 PM" src="https://user-images.githubusercontent.com/23002114/86534880-bb7eb980-bef9-11ea-97c3-fd0561bce8e5.png">